### DISCUSSION TITLE
Nous a1, weird version, I couldn't find proper info

### DISCUSSION THREAD

**Discussion by zillion42** [arendst/Tasmota#17209] [General]
I have bought some nous a1 which actually have some kind of esp, according to mac address. I spent quite some time trying to open it in all the wrong ways, I finally managed and then found this site...
https://kriwanek.de/index.php/de/homeautomation/esp8266-ger%C3%A4te/andere-ger%C3%A4te/576-neo-coolcam-smart-plug
So I followed the instruction as best I could, desoldering the 6 contacts to reach the backside. Getting there I abused and broke the board quite a bit. I didn't try to flash it, because I am unsure which pin the GPIO is. Instead I put everything together again, resoldered the 6 pins and the broken piece and to my surprise it still works, with tuya app.
My questions now, what is that long thing that I broke out, where is the GPIO pin ? Will this work with the Neo Coolcam profile ? Thx in advance.
![_DSC0352 copy](https://user-images.githubusercontent.com/36650866/204391964-f74d95b6-9713-4af7-8bc5-171938d9b288.jpg)
![20221128_193455 copy](https://user-images.githubusercontent.com/36650866/204391968-47c7a2bd-6d20-41ed-a0eb-b3ce447a5f4d.jpg)
![20221128_193513 copy](https://user-images.githubusercontent.com/36650866/204391969-8fd70b2c-a4e2-417a-8bbf-22a8a9f6375b.jpg)
![20221128_193519 copy](https://user-images.githubusercontent.com/36650866/204391975-50b75025-5755-4ab0-83db-c2f285407630.jpg)
![_DSC0349 copy](https://user-images.githubusercontent.com/36650866/204391979-e5129cbc-f54a-4df3-bcf5-779c2be13b0f.jpg)

**Comment by sfromis** [comment]
I see a Beken logo there, meaning not Tasmota-compatible. The variant working with Tasmota should be the Nous A1T.

**Reply by zillion42** [reply]
aww, that's unfortunate, again.
What does this mean ?
![2022_11_28_23_43_47_FRITZ_Box_6591_Cable](https://user-images.githubusercontent.com/36650866/204396701-5074134b-96a6-4e08-9298-37fdef8206f8.jpg)
![2022-11-28 23_46_28-MAC Address Lookup - MAC Lookup Online](https://user-images.githubusercontent.com/36650866/204396959-d8f3568e-a229-4a80-9ba9-cae70e7300b3.png)

**Reply by zillion42** [reply]
I guess you would reply, it's officially espressif vendor and it has an esp chip, but still. its not tasmota compatible. I think I get it.

**Reply by sfromis** [reply]
Depends on the context, but it means that you have some Espressif broadcasting an AP name. It is not always clear which device it is.

**Reply by sfromis** [reply]
No, that's not what I would reply. The chip in your image is from Beken and absolutely not Espressif. The AP name you look at is likely from some other device in the vicinity.

**Reply by zillion42** [reply]
I guess you're right, my bad, it is in fact CC-8C-BF
![2022-11-29 00_12_45-MAC Address Lookup - MAC Lookup Online](https://user-images.githubusercontent.com/36650866/204400306-38a2cb08-6a12-4b98-8d71-8ab9c56d8f70.png)

I'm really wasting my time, and I am very unlucky with the stuff I order. I have nous a1t coming now, quite expensive though.

**Reply by sfromis** [reply]
Your existing plug may not be a waste, you may have luck with https://github.com/kuba2k2/libretuya/ or https://github.com/openshwprojects/OpenBK7231T_App/

**Reply by zillion42** [reply]
thx
